conda/plugins/hookspec.py

```diff
@@ -29,9 +29,12 @@ def conda_solvers(self) -> Iterable[CondaSolver]:
         Example:
 
         .. code-block:: python
-
             from conda import plugins
             from conda.core import solve
+            import logging
```

Comment:
```suggestion
            import logging
            
            from conda import plugins
            from conda.core import solve
```

Reply:
Thank you for this suggestion @jezdez ! Looks like the import statements getting re-arranged enabled all of the original whitespace to pass linter checks. 👍